db/data/20211112120154_remove_trainee_and_update_others.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class RemoveTraineeAndUpdateOthers < ActiveRecord::Migration[6.1]
+  def up
+    Trainee.find_by(trainee_id: "TI00287920").destroy # DTTP record has been removed
```

Comment:
This might update/destroy an unexpected record since trainee ids are not necessarily unique, hence it might be safer to use slugs/register (trainee) ids.

Also, in terms of updating the TRN, that should _have happened automatically&trade;_, which suggest that it might need some sort updating of the dttp/dttp placement ids (but I'm speculating on that point)
Once that is done, the trn can be retreived using this job https://github.com/DFE-Digital/register-trainee-teachers/blob/main/docs/support_playbook.md#dttpretrievetrnjob-for-trainee-id--has-timed-out-after-4-days

Reply:
The requirements threw me off track and after looking at the data in prod, I see what's actually going on now.

So we have duplicate records for trainee ID: `TI00287920` and `TI287849`. The latest records have a TRN already which means we don't have to worry about that now. The older versions still need to be removed, so the PR has been updated to use the slug value to find and remove these records.